src/helpers/render-document-file.js

```diff
@@ -19,6 +19,7 @@ import { vNodeHasChildren } from '../utils/vnode';
 import { isValidUrl } from '../utils/url';
 import { getMimeType } from '../utils/image';
 import { cloneDeep } from 'lodash';
+import { buildHorizontalRule } from '../helpers/xml-builder';
```

Comment:
although I like this syntax better, for now to keep it consistent, follow the syntax of how the xmlBuilder is imported on line 15. 

EG `xmlBuilder.buildHorizontalRule` in the code below